### DISCUSSION TITLE
Can we use this EDK II(UEFI) firmware directly on the Physical board?

### DISCUSSION THREAD

**Discussion by Rushikesh3499** [tianocore/edk2#2922] [General]
Hi,
    Actually I want to use this UEFI firmware which is created using EDK II source code directly on the bare metal. Is it possible? 
If yes then how to do that and if no then what other requirements to do that?
    And also there is compulsion to add the SCP binaries and device tree binaries and certificates to make the fip.bin file or only by giving the BL2.bin and BL31.bin is Ok?

**Comment by spbrogan** [comment]
edk2 itself only contains the code to create usable images for a few virtual platforms.  

Edk2-platforms contains platform code that when used with edk2 can create images for additional platforms.  

Not sure what physical board you are using but it is unlikely unless you bought one of the few boards supported by open source.
